apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    IDetailQueryResult,
+    ITranscribedAudioViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { Link } from 'react-router-dom';
+import { routes } from '../../../app/routes/routes';
+import './transcribed-audio-detail.thumbnail.presenter.css';
+
+export const TranscribedAudioDetailThumbnailPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const location = `/${routes.resources
+        .ofType(ResourceType.transcribedAudio)
+        .detail(transcribedAudioItem.id)}`;
+
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
+
+    return (
+        <div className="detail-thumbnail-container">
+            <Link to={location}>
+                <div
+                    className="detail-thumbnail-meta-container"
+                    title="View Connected Transcribed Audio"
+                >
+                    <strong>Transcribed Audio ID:</strong> {transcribedAudioItem.id}
+                    <br />
+                    <strong>Run Time:</strong> {runTime} Sec
```

Comment:
Also, at a practical level, it would be less reading \ writing to use a map or reduce to wrap this in.

Reply:
We may write a wrapping function to add the `<br />` tag as necessary